inc/enqueue.php

```diff
@@ -90,12 +90,9 @@ function whichHeader() {
  * @since 1.0
  */
 if ( ! function_exists( 'largo_footer_js' ) ) {
-	function largo_footer_js() { ?>
+	function largo_footer_js() {
 
-
-		<?php
-		// Are the widgets that contain facebook social buttons loaded (or are we on single/author?)
-		if( largo_facebook_widget::is_rendered() || largo_follow_widget::is_rendered() || is_single() || is_author() ) : ?>
+		if( largo_facebook_widget::is_rendered() || largo_follow_widget::is_rendered() || is_single() || is_author() ) : ?>		
```

Comment:
Do we need the checks for `is_single()` and `is_author()` here if the scripts are no longer used as a result of the page being an author or single page?